job_spec/OPERA_DIST_S1.yml

```diff
@@ -0,0 +1,96 @@
+OPERA_DIST_S1:
+  required_parameters:
+    - mgrs_tile_id
+    - track_number
+    - post_date
+  parameters:
+    mgrs_tile_id:
+      api_schema:
+        description: MGRS tile ID
+        minLength: 5
+        maxLength: 5
+        type: string
+        example: "11SLT"
+    post_date:
+      api_schema:
+        description: Post acquisition date
+        type: string
+        maxLength: 10
+        minLength: 10
+        pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
+        example: "2025-01-21"
+    track_number:
+      api_schema:
+        description: "Sentinel-1 Track Number; Supply one from the group of bursts collected from a pass; Near the dateline you may have two sequential track numbers but only need one"
+        type: integer
+        default: 1
+        minimum: 1
+        maximum: 175
+        example: 71
+    memory_strategy:
+      api_schema:
+        description: Memory strategy to use for GPU inference
+        type: string
+        enum: ["high", "low"]
+        default: "low"
+        example: "high"
```

Comment:
No need to provide `example`, although I don't think it hurts. In the past I've only provided `default` for non-required parameters and it populates into the example job in the swagger UI.

If you want to keep both, then for `memory_strategy` specifically, I suspect we'd want the example to match the default?
```suggestion
```